QtNodeEditor/src/NodeGraphicsObject.cpp

```diff
@@ -367,6 +371,11 @@ hoverEnterEvent(QGraphicsSceneHoverEvent * event)
   // bring this node forward
   setZValue(1.0);
 
+
+//  node().mo
+
+
+
```

Comment:
these lines should be removed...

Reply:
There are also some unused commented code in `mainwindow.cpp` that should be removed